packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.tsx

```diff
@@ -33,6 +34,9 @@ export class GuxPaginationCursor {
   @Prop()
   hasNext: boolean = false;
 
+  @Prop()
+  label: string;
```

Comment:
What is the benefit of passing in a label to assign an aria-label to a container in the shadowdom? Is adding an aira-label to the host component not the same?

Reply:
@daragh-king-genesys The label needs to be on the the element that creates the landmark, the `<nav>` in this case.